packages/editor/src/components/block-edit/edit.native.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { withFilters } from '@wordpress/components';
+import { getBlockType } from '@wordpress/blocks';
+
+export const Edit = ( props ) => {
+	const { attributes = {}, name } = props;
+	const blockType = getBlockType( name );
+
+	if ( ! blockType ) {
+		return null;
+	}
+
+	const style = props.style || attributes.style;
+	const Component = blockType.edit;
+
+	return (
+		<Component
+			{ ...omit( props, [ 'className' ] ) }
```

Comment:
It's interesting because `blocks` package doesn't define any styles. It tries to be fully UI agnostic. So I guess it only passes down those class names. It seems like `edit` implementations provided for RN should simply ignore it instead.

Reply:
Makes sense 👍